Docs update for the latest version of azure-maps-control

S-J-M opened the pull request.

v-zazhou committed:
Replace legacy build pipeline with SDP build pipeline (#71)

yunair committed:
Merge branch 'master' into smoke-test

Commit:
CI Update
 Build.Reason:Manual by Zaofeng Zhou
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=165420&view=results

Commit:
CI Update
 Build.Reason:Manual by Christian Gennaro
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=231510&view=results

opbld31 commented:
Docs Build status updates of commit _[0b2dd58](https://github.com/MicrosoftDocs/azure-iot-docs-sdk-typescript/commits/0b2dd5811629d016e84a258bcad4ebda3bea9308)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-9a3e03c5)

<a id="9a3e03c5"></a>
###
 - **[Error-PullRequestMergeConflict]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C7%5C13%5C8a959d8c-500f-0fdb-1554-1e193b0f705a%5CPullRequest%5C202107132043113288-79%5Cworkflow_report.html?skoid=5f458e1a-fa67-4ae6-a972-54ea42ec670e&sktid=975f013f-7f24-47e8-a7d3-abc4752bf346&skt=2021-07-13T08%3A15%3A34Z&ske=2021-07-20T08%3A20%3A34Z&sks=b&skv=2020-04-08&sv=2020-04-08&st=2021-07-13T20%3A38%3A41Z&se=2021-08-13T20%3A43%3A41Z&sr=b&sp=r&sig=gYFyC45HgpjvxrA6lfJwSp3Hmwm2HfLo8XJhxgs4%2B70%3D).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8a959d8c-500f-0fdb-1554-1e193b0f705a?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

Commit:
CI Update
 Build.Reason:Manual by Shubha Jain
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=237708&view=results

opbld31 commented:
Docs Build status updates of commit _[88da50f](https://github.com/MicrosoftDocs/azure-iot-docs-sdk-typescript/commits/88da50f31042cd52222dc8ee93916a69001194b2)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-b4616f34)

<a id="b4616f34"></a>
###
 - **[Error-PullRequestMergeConflict]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C7%5C27%5C8a959d8c-500f-0fdb-1554-1e193b0f705a%5CPullRequest%5C202107271726527128-79%5Cworkflow_report.html?skoid=5f458e1a-fa67-4ae6-a972-54ea42ec670e&sktid=975f013f-7f24-47e8-a7d3-abc4752bf346&skt=2021-07-26T04%3A14%3A40Z&ske=2021-08-02T04%3A19%3A40Z&sks=b&skv=2020-04-08&sv=2020-04-08&st=2021-07-27T17%3A22%3A23Z&se=2021-08-27T17%3A27%3A23Z&sr=b&sp=r&sig=Ehrui%2BIXTDKhBxzip3UdKy2PCrBtYNBdUNwKxrH1g%2Bw%3D).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8a959d8c-500f-0fdb-1554-1e193b0f705a?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

Commit:
CI Update
 Build.Reason:Manual by Robert Outlaw
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=240679&view=results

opbld31 commented:
Docs Build status updates of commit _[83656ce](https://github.com/MicrosoftDocs/azure-iot-docs-sdk-typescript/commits/83656ced1419b4451b5659bf315bdec248b6254b)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-d0912d22)

<a id="d0912d22"></a>
###
 - **[Error-PullRequestMergeConflict]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C7%5C29%5C8a959d8c-500f-0fdb-1554-1e193b0f705a%5CPullRequest%5C202107291911485953-79%5Cworkflow_report.html?skoid=5f458e1a-fa67-4ae6-a972-54ea42ec670e&sktid=975f013f-7f24-47e8-a7d3-abc4752bf346&skt=2021-07-28T03%3A46%3A06Z&ske=2021-08-04T03%3A51%3A06Z&sks=b&skv=2020-04-08&sv=2020-04-08&st=2021-07-29T19%3A07%3A15Z&se=2021-08-29T19%3A12%3A15Z&sr=b&sp=r&sig=6Ur6pFL47SMHd33f65YjoA7tpqAWuVyLwHqivTps48Q%3D).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8a959d8c-500f-0fdb-1554-1e193b0f705a?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>
